README.md

```diff
@@ -0,0 +1,42 @@
+# ftrack OpenAssetIO Plugin
+
+A skeleton OpenAssetIO ManagerPlugin that binds ftrack to the API
+
+> **Warning**
+> This plugin is highly experimental, and subject to significant
+> change.
+
+## Getting started
+
+To use the plugin in an OpenAssetIO host, set (or append) the
+`OPENASSETIO_PLUGIN_PATH` env var to include the `python` directory in
+this checkout.
+
+The plugin requires `openassetio` to be available to python at runtime
+(see the [project documentation](https://github.com/OpenAssetIO/OpenAssetIO#getting-started)
+for more information).
+
+The plugin provides a manager with the identifier `com.ftrack`.
```

Comment:
[consider] Users are likely to skip the "Testing" section, but currently you have to read it or you may not know to `pip install -r requirements.txt`. Perhaps best to mention `requirements.txt` here?  Or is it expected that users will already have those dependencies  in their environment?

Reply:
See revised documentation [here](https://github.com/foundrytom/OpenAssetIO-Manager-ftrack/pull/1/commits/60cc8be3ed0bf89fc2c18547836d3171f2654a56).